monitor/service.go

```diff
@@ -163,28 +163,17 @@ type RemoteWriterConfig struct {
 	ClusterID  uint64
 }
 
-// SetRemoteWriter can be used via and RPC call to set a remote location
-// for writing monitoring information.
-func (m *Monitor) SetRemoteWriter(c RemoteWriterConfig) error {
+// SetPointsWriter can be used to set a remote location for writing monitoring
+// information. This will use the default interval and database.
+func (m *Monitor) SetPointsWriter(pw PointsWriter) error {
 	// Ignore the monitor's config settings.
 	m.mu.Lock()
 	m.storeEnabled = true
```

Comment:
I think we should put back the ability to control the config options, while you're in this bit of the code-base. We can return early if the store is disabled.